packages/discovery-provider/src/utils/redis_constants.py

```diff
@@ -43,13 +43,25 @@
 )
 latest_sol_user_bank_backfill_db_tx_key = "latest_sol_program_tx:user_bank_backfill:db"
 
+latest_sol_payment_router_program_tx_key = "latest_sol_program_tx:payment_router:chain"
+latest_sol_payment_router_db_tx_key = "latest_sol_program_tx:payment_router:db"
+
+latest_sol_payment_router_backfill_program_tx_key = (
+    "latest_sol_program_tx:payment_router_backfill:chain"
```

Comment:
do we need these backfill keys? i think i made these for the audio transactions history backfilling project, don't think they're used anymore and i could probably remove those. but don't think we need them for payment router?